packages/web/src/components/comments/CommentBlock.tsx

```diff
@@ -52,6 +55,16 @@ const CommentBlockInternal = (
 ) => {
   const { comment, parentCommentId, isPreview } = props
   const { track, artistId } = useCurrentCommentSection()
+  const isDarkMode = getIsDarkMode()
+
+  // Get the commentId query param from the url
+  const [searchParams] = useSearchParams()
+  const commentIdParam = searchParams.get('commentId')
+  const { data: highlightComment } = useComment(HashId.parse(commentIdParam))
```

Comment:
we should make sure that useComment is a cache hit for all comments that are currently rendered. i think we do prime the comment cache in useComments right?

Also can we make this a separate hook for readability, like `useHighlightedComment`?